okhttp/src/main/java/okhttp3/internal/http1/Http1Codec.java

```diff
@@ -74,6 +74,7 @@
   private static final int STATE_OPEN_RESPONSE_BODY = 4;
   private static final int STATE_READING_RESPONSE_BODY = 5;
   private static final int STATE_CLOSED = 6;
+  private static final int HEADER_LIMIT = Integer.getInteger("okhttp.headerlimit", 256 * 1024);
```

Comment:
Not sure how I feel about making this configurable. In particular I don’t like the idea of anyone actually configuring this!

Reply:
Do you want to remove it.  I think OkHttp is an opinionated client library, not a toolkit that must be infinitely configurable.  And this shouldn't have a public API.

But as I expected someone will find this breaks their awkward API, this might be a workaround.

Do you want me to remove it?

Reply:
Yeah, please do. No urgency though!

Reply:
https://github.com/square/okhttp/pull/3613